### ISSUE TITLE
Text Display Entities Transparency not working

### ISSUE THREAD

**Issue by DrBiznes** [X0nk/Bliss-Shader#346]
### Describe the bug
![2024-11-20_16 37 29](https://github.com/user-attachments/assets/1cc2b6ec-0a99-4f73-9146-6488bbec74f1)
Text Display Entities should be transparent but have a white background instead. Lot's of new servers are using Text Display Entities in place of the old text holograms and this shader pack is practically unusable on those servers now.

### Describe how your minecraft is set up
1. Include a screenshot of the in-game f3 debug overlay. Please make sure it is fully visible.
![2024-11-20_16 38 56](https://github.com/user-attachments/assets/47239554-e0ed-4af6-a181-f262a5cdeffb)

2. If you cannot do option 1. Please list the following information instead:
- What GPU vendor you are using (e.g Apple/AMD/Nvidia/Intel)
- What operating system you are using (e.g Windows/Linux/macOS)
- What minecraft version you are playing (e.g 1.20.4, 1.16.5, 1.12)
- If you are using a special client for minecraft (e.g lunar, badlion, feather)
- What shader loading mod you are using (e.g Iris/oculus/optifine)
- What version of the shader you are using (e.g Bliss v2.0.4, Bliss Shader Main)
- If you are using any, what additional mods you are using besides the ones required to use the shader (e.g distant horizons, physics mod)

### Describe how to reproduce the bug
Show exactly how to make the bug appear, if you can. Video footage is preferred.

IF you want to recreate this bug in single player just summon a text display entity in game with a transparent background.
"The background color, arranged by ARGB. Since pixel with an alpha channel less than 0.1 are discarded when rendering in vanilla [shader](https://minecraft.wiki/w/Shader), the background becomes fully transparent when A is less than 26 (0x1A). Defaults to 1073741824 (0x40000000). Interpolated." Taken from [the wiki](https://minecraft.wiki/w/Display)

Isolate the bug. some multiplayer servers cause issues on their own. in a newly made singleplayer world, with default/reset shader settings, show the steps to cause the bug

if it is happening on content from a mod, list what mod, and what content of that mod it happens on.

if a specific shader setting can make it start/stop bugging, please list what shader setting does that.

**Comment by neighborsbear** [comment]
<img width="1920" height="1017" alt="Image" src="https://github.com/user-attachments/assets/b162f09d-8768-484a-8dbd-dfb2f74a838b" />

<img width="1920" height="1017" alt="Image" src="https://github.com/user-attachments/assets/28d75355-1ea5-47f4-a7d8-19560567cfb3" />

There are still a lot of problems

1.21.1
2.1.2

**Comment by X0nk** [comment]
this is probably not the same problem. this is most likely because i set `shader settings -> debug/experimental -> TRANSLUCENT_ENTITIES` to OFF by default in bliss v2.1.2 due to issues with it becoming too large to ignore in mc 1.21.10

**Comment by NightmareWuxin** [comment]
However, if i enable that option, the translucent entities become from black to non-translucent and the name tag still black shade

**Comment by wouo46** [comment]
i have the exact same issue, both on servers and on singleplayer name tags and text display entities have a black background that i cant get rid of; a shame cuz its a really nice shaderpack but with this issue its unplayable
edit: i turned translucent entities on in experimental settings and now the background is white

**Comment by Gavn007** [comment]
Hey i made a fix for this glitch in my version of bliss for me and my friends modpack 

TLDR:
1. The translucent entity pass forced `alpha = 1.0`, turning the nametag's semi-transparent background plate into a solid black rectangle covering the text
2. World lighting dimmed nametag text with time-of-day (gray at night, black in caves)
3. Auto-exposure multiplied the whole screen including nametags, pulling white text down to gray
4. AgX tonemapper maps raw white (1.0) to display luma ~0.79 (gray, not white)
5. Glass-Tint in the composite pass normalized the plate's dark color to mid-gray and tinted the world behind it
6. Refraction warped nametag texcoords, subtly blurring glyphs
7. Border fog desaturated distant nametags toward gray
